app/controllers/collections_controller.rb

```diff
@@ -3,4 +3,28 @@ def index
         collections = Collection.all 
         render json: collections
     end
+
+    def create
```

Comment:
You've got create method that's not a part of the trello card corresponding. Should be removed.